src/pages/debrief/debrief.selector.ts

```diff
@@ -17,10 +17,31 @@ export const getSeriousOrDangerousFaults = (faults: SeriousFaults | DangerousFau
 export const getDrivingFaults = (faults: DrivingFaults): FaultCount[] => {
   const faultsEncountered : FaultCount[] = [];
   forOwn(faults, (value, key) => {
-    if (value > 0) {
+    if (value > 0 && !key.endsWith('Comments')) {
       const label = key as keyof typeof competencyLabels;
-      faultsEncountered.push({ name: fullCompetencyLabels[label], count: value });
+      faultsEncountered.push({ propertyName: key, name: fullCompetencyLabels[label], count: value });
     }
   });
   return faultsEncountered.sort((a, b) => b.count - a.count);
 };
+
+export const displayDrivingFaultComments = (data: TestData) : boolean => {
+  const seriousFaults =  getSeriousOrDangerousFaults(data.seriousFaults);
```

Comment:
Why don't we have 2 separate functions, `getSeriousFaults` and `getDangerousFaults`?

Reply:
Implemented in mes-2033.